lib/apkstats/command/apk_analyzer.rb

```diff
@@ -59,6 +73,15 @@ def self.to_feature(str)
       ::Apkstats::Entity::Feature.new(name, not_required: kind == "not-required", implied_reason: kind == "implied:" && tail)
     end
 
+    def self.parse_reference_to_map(command_output)
+      reference_map = {}
+      command_output.split(/\r?\n/).map do |s|
+        dex_file, method_count = s.strip.split(/\t/)
```

Comment:
Ruby's tuple assignment works like below.

```
arr = ['x', 'y', 'z']
x, y = arr
puts x #=> 'x'
puts y #=> 'y'
```

If this requires only 1st and 2nd elements, then please define like `dex_file, method_count, = s.strip.split(/\t/)` to emphasize the meaning.
Otherwise, could you please specify the split count?

Reply:
I'll do specify the split count.